confirm.go

```diff
@@ -104,8 +104,14 @@ func (c *Confirm) getBool(showHelp bool) (bool, error) {
 	return c.Default, nil
 }
 
-// Prompt prompts the user with a simple text field and expects a reply followed
-// by a carriage return.
+/*
+Prompt prompts the user with a simple text field and expects a reply followed
+by a carriage return.
+
+	likesPie := false
+	prompt := &survey.Confirm{ Message: "What is your name?" }
+	survey.AskOne(prompt, &name, nil)
```

Comment:
This one looks like a bad copy/paste, correct the prompt to something like `Do you like Pie?`, and use the `likePies` variable.